sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuWindowExec.scala

```diff
@@ -753,6 +754,14 @@ object GroupedAggregations {
     }
   }
 
+  private def getMinPeriodsFor(boundGpuWindowFunction: BoundGpuWindowFunction): Int =
+    boundGpuWindowFunction.windowFunc match {
+      case GpuCount(_, _) => 0
```

Comment:
Can we make this a part of `GpuWindowFunction` instead? I don't really like having special code that just knows how to handle specific aggregation functions. It makes it harder to maintain the code.

Reply:
Oh, right. Thanks, that is much better.

Reply:
Moved.